### ISSUE TITLE
[css-typed-om] Add a note about partial implementations?

### ISSUE THREAD

**Issue by darrnshn** [w3c/css-houdini-drafts#733]
Originally raised by @lilles. We should add a note about partial implementations, whether they are permitted, and how they will work. We should also mention something about how web developers can write code that will be compatible across different Typed OM levels or partial implementations (if permitted).

**Comment by tabatkins** [comment]
Yeah, that seems valuable, particularly since this spec will be updated over time as new properties get added or older properties get modified .
